src/site/stages/build/drupal/graphql/vetCenterLocations.graphql.js

```diff
@@ -0,0 +1,231 @@
+        entityBundle
+        fieldAddress {
+          locality
+          administrativeArea
+          postalCode
+          addressLine1
+        }        
+        fieldPhoneNumber
+        fieldMedia {
+          entity {
+            ... on MediaImage {
+              image {
+                alt
+                title
+                derivative(style: _32MEDIUMTHUMBNAIL) {
+                  url
+                  width
+                  height
+                }
+              }
+            }
+          }
+        }                             
+      }          
+      ... on NodeVetCenterOutstation {
+        entityBundle
+        fieldAddress {
+          locality
+          administrativeArea
+          postalCode
+          addressLine1
+        }
+        fieldPhoneNumber
+        fieldMedia {
+          entity {
+            ... on MediaImage {
+              image {
+                alt
+                title
+                derivative(style: _32MEDIUMTHUMBNAIL) {
+                  url
+                  width
+                  height
+                }
+              }
+            }
+          }
+        }              
+      }
+      ... on NodeVetCenterCap {
+        entityBundle
+        fieldAddress {
+          locality
+          administrativeArea
+          postalCode
+          addressLine1
+        }        
+        fieldMedia {
+          entity {
+            ... on MediaImage {
+              image {
+                alt
+                title
+                derivative(style: _32MEDIUMTHUMBNAIL) {
+                  url
+                  width
+                  height
+                }
+              }
+            }
+          }
+        }                            
+      }
+      ... on NodeVetCenterMobileVetCenter {
+        entityBundle              
+        fieldAddress {
+          locality
+          administrativeArea
+          postalCode
+          addressLine1
+        }      
+        fieldPhoneNumber
+        fieldMedia {
+          entity {
+            ... on MediaImage {
+              image {
+                alt
+                title
+                derivative(style: _32MEDIUMTHUMBNAIL) {
+                  url
+                  width
+                  height
+                }
+              }
+            }
+          }
+        }                              
+      }              
+    } 
+  }
+  fieldOffice {
+    entity {
+      ... on NodeVetCenter {
+        reverseFieldOfficeNode(limit: 500, filter: {conditions: [{field: "type", value: ["vet_center_outstation", "vet_center_cap", "vet_center_mobile_vet_center"]}]}) {
+          entities {
+            ... on NodeVetCenterCap {
+              title
+              entityBundle
+              fieldMedia {
+                entity {
+                  ... on MediaImage {
+                    image {
+                      alt
+                      title
+                      derivative(style: _32MEDIUMTHUMBNAIL) {
```

Comment:
just wondering - how did you land on using `_32MEDIUMTHUMBNAIL` as the value? was that in the GraphQL Explorer?

Reply:
It was in GraphQL Explorer, and we were told to use it by the CMS team during an interactive coding session.